pkgvars

```diff
@@ -1,4 +1,6 @@
 pkgname=citus
 pkgdesc='Citus (Open-Source)'
-pkglatest=10.2.3.citus-1
+pkglatest=10.1.4.citus-1
+releasepg=11,12,13
+nightlypg=12,13
```

Comment:
I do not think these fields make sense.

1. Do we need `releasepg` here?
2. Do we need `nightlypg` here?
3. PG11 support was dropped in `10.1` and we should not have that in this field here.

Reply:
I missed that part as well. I deleted this part as well. Thanks for the comment